src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -796,6 +834,15 @@ const NetworkModificationNodeEditor = () => {
                     onClick={toggleSelectAllModifications}
                 />
                 <div className={classes.filler} />
+                <IconButton
+                    className={classes.toolbarIcon}
+                    color="primary"
```

Comment:
Remove the color. This button should be white.

Reply:
done